gdscript-docs.el

```diff
@@ -35,16 +35,17 @@
 (require 'gdscript-customization)
 
 ;;;###autoload
-(defun gdscript-docs-browse-api ()
-  "Open the main page of Godot API in eww browser."
+(defun gdscript-docs-browse-api (&optional force-online)
+  "Open the main page of Godot API in eww browser. prefix (C-u) to force online"
```

Comment:
The docstring should be like `Use the universal prefix (C-u) to force browsing the online API.`

Reply:
Agreed